src/main/webapp/app/quiz-statistics/drag-and-drop-question-statistic/drag-and-drop-question-statistic.component.ts

```diff
@@ -363,6 +367,15 @@ export class DragAndDropQuestionStatisticComponent implements OnInit, OnDestroy,
         }
     }
 
+    resizeImage() {
+        setTimeout(() => {
+            const image = document.querySelector('.drag-and-drop-quizStatistic-picture jhi-secured-image img') as HTMLImageElement;
+            const clickLayer = document.getElementsByClassName('click-layer').item(0) as HTMLElement;
+            clickLayer.style.width = image.width + 'px';
+            clickLayer.style.height = image.height + 'px';
+        }, 500);
```

Comment:
Why is there a timeout of 500ms? If this is a valid implementation detail, it should be put into JSDoc.